cmd/probe/src/lib.rs

```diff
@@ -240,13 +241,13 @@ fn probecmd(
             //
             let rval = core
                 .halt()
-                .and_then(|_| core.read_reg(ARMRegister::PC))
+                .and_then(|_| core.read_reg(Register::Arm(ARMRegister::PC)))
```

Comment:
Shouldn't this use the `get_pc` method?

Reply:
This command has not yet been ported to riscv, so this is just a quick fix to maintain arm functionality.

Reply:
Ack